File: Python/ex20-Pintando a parede.py
import math
l = float(input())
a = float(input())
c = float(input())
m = float(input())

area = l * a

quantidade = area / m

custo = math.ceil(quantidade) * c

print('{:.0f}'.format(math.ceil(quantidade)))
print('{:.2f}'.format(custo))

File: exercicios_e_aulas/ex20-pintando_a_parede.c
#include<stdio.h>
#include<math.h>

float area(float l, float h){
    float area = l * h;

    return area;
}

int quantidade(float r, float l, float h){
    float a = area(l, h);
    int quant = ceil(a / r);

    return quant;
}

float custo(int q, float c){
    float price = q * c;

    return price;
}


int main(int argc, char *argv[]){
    float l;
    scanf("%f", &l);
    float h;
    scanf("%f", &h);
    float c;
    scanf("%f", &c);
    float r;
    scanf("%f", &r);

    int latas = quantidade(r, l, h);
    float price = custo(latas, c);

    printf("%d\n%.2f", latas, price);
}